redpanda/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,42 @@
+name: Redpanda
+files:
+- name: redpanda.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - name: use_openmetrics
+      description: |
+        Use the latest OpenMetrics implementation for more features and better performance.
+        If running on Agent v6, you must set the `python_version` option in `datadog.yaml` to `"3"`.
+      value:
+        example: true
+        display_default: false
```

Comment:
Just to confirm, is it intentional that `display_default` is set to `false`, whereas the `example` is set to `true`? This sets the option header in the `conf.yaml.example` to display `default: false`: https://github.com/DataDog/integrations-extras/pull/1020/files#diff-aeabab6138b04e7abd3938c290d6aae0de65361c4d7145b1eb8f48c0b0aa2dfeR53
If the default value should be `true`, you can remove `display_default: false`.

Reply:
Thanks! I removed the `display_default: false`.